sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/AbstractTypesShouldNotHaveConstructors.cs

```diff
@@ -35,6 +35,8 @@ public class AbstractTypesShouldNotHaveConstructors : SonarDiagnosticAnalyzer
     {
         internal const string DiagnosticId = "S3442";
         private const string MessageFormat = "Change the visibility of this constructor to 'protected'.";
+        
+        public string Bar = "Foo";

```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this field 'private' and encapsulate it in a 'public' property. ([csharpsquid:S1104](https://sonarcloud.io/organizations/sonarsource/rules#rule_key=csharpsquid%3AS1104))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=sonaranalyzer-dotnet&pullRequest=1344&issues=AWLia6xcjr7wspRo5hSR&open=AWLia6xcjr7wspRo5hSR)

[ISSUE_KEY]: AWLia6xcjr7wspRo5hSR